src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/components/dynamic-attribute-control/dynamic-attribute-control.component.html

```diff
@@ -0,0 +1,18 @@
+<div *ngIf="!!attributesForm" [formGroup]="attributesForm">
+	<label [attr.for]="attribute.key">{{ attribute.label }}</label>
+
+	<div [ngSwitch]="attribute.type">
+		<textarea *ngSwitchCase="controlType.FreeText" [formControlName]="attribute.key" [id]="attribute.key" [value]="attribute.value"></textarea>
+		<select *ngSwitchCase="controlType.SingleChoice" [id]="attribute.key" [formControlName]="attribute.key" [value]="attribute.value">
+			<option *ngFor="let opt of attribute.options">{{ opt.value }}</option>
```

Comment:
is there a value attribute missing? for ex) `<option *ngFor="..." [value]="opt.value">{{opt.value}}</option>`